packages/@webex/plugin-meetings/test/unit/spec/meeting-info/index.js

```diff
@@ -7,6 +7,8 @@ import MeetingInfo from '@webex/plugin-meetings/src/meeting-info/index';
 import MeetingInfoUtil from '@webex/plugin-meetings/src/meeting-info/util';
 import MeetingInfoRequest from '@webex/plugin-meetings/src/meeting-info/request';
 
+const flushPromises = () => new Promise(setImmediate);
```

Comment:
there is already flushPromises in packages/@webex/plugin-meetings/test/utils/testUtils.js